CodeStarterCamp_Week2/main.swift

```diff
@@ -5,8 +5,35 @@
 //  Created by yagom.
 //  Copyright © yagom academy. All rights reserved.
 //
-
 import Foundation
 
-print("Hello, World!")
+func createLottoNumbers() -> [Int] {
+    var lottoNumbers: [Int] = []
+    var index: Int = 0
+
+    while index < 6 {
+        let randNum: Int = Int(arc4random_uniform(45)) + 1
+
+        if lottoNumbers.firstIndex(of: randNum) == nil {
+            lottoNumbers.append(randNum)
+            index = index + 1
+        }
+    }
+
+    return lottoNumbers
+}
+
+func confirmLottoNumbers(_ myLottoNumbers:[Int], _ createdLottoNumbers:[Int]) -> String {
+    let winNumbers: [Int] = myLottoNumbers.filter{createdLottoNumbers.contains($0)}
```

Comment:
보통..
```suggestion
    let winNumbers: [Int] = myLottoNumbers.filter{ createdLottoNumbers.contains($0) }
```
이렇게  쓰곤 합니다. 

축약형을 사용해주셨는데, 이건 어떻게 풀어쓸 수 있을까요?